keel-actuator/src/test/kotlin/com/netflix/spinnaker/keel/actuation/ResourcePersisterTests.kt

```diff
@@ -111,12 +111,17 @@ internal class ResourcePersisterTests : JUnit5Minutests {
         context("after an update") {
           before {
             resourcesDueForCheck()
-            update(DummyResourceSpec("kthxbye"))
+            subject.upsert(SubmittedResource(
+              metadata = SubmittedMetadata("keel@spinnaker"),
```

Comment:
I think `SubmittedMetadata` can be done away with now the `Resource` class just uses a `Map<String, Any?>`.

Reply:
I had some trouble getting the auth function to work with a map. Overall, it seemed simpler to just have this data class.